### PR TITLE
Minor code changes / removed a lot a redundant stuff

### PR DESCRIPTION
**Pull request by BramEsendam**

### CODE REVIEW

File: MonkeyBot/CommandManager.cs [hunk 1]
```diff
@@ -155,12 +155,12 @@ private string GetCommandErrorMessage(CommandError error, string prefix, string
 
         public async Task BuildDocumentationAsync()
         {
-            string docuHTML = DocumentationBuilder.BuildDocumentation(commandService, DocumentationOutputTypes.HTML);
-            string fileHTML = Path.Combine(AppContext.BaseDirectory, "documentation.html");
+            var docuHTML = DocumentationBuilder.BuildDocumentation(commandService, DocumentationOutputTypes.HTML);
```
Comment (MarkusKgit) [inline MonkeyBot/CommandManager.cs]: I'd like to stick to the corefx coding style regarding var

> We only use var when it's obvious what the variable type is (e.g. var stream = new FileStream(...) not var stream = OpenStandardInput()).

Could you kindly either revert the commit or manually only assign var to places where the type will be obvious? I can also do it based on this commit of course

File: MonkeyBot/Common/XYChart.cs [hunk 2]
```diff
@@ -50,9 +50,9 @@ public void ExportChart(string filePath, IEnumerable<PointF> xyValues)
                 throw new ArgumentException("Please provide some values");
 
             using (Image image = new Bitmap(chartWidth, chartHeight))
-            using (Graphics graphics = Graphics.FromImage(image))
-            using (AdjustableArrowCap arrowCap = new AdjustableArrowCap(5, 5))
-            using (Pen axisPen = new Pen(Brushes.Black, 5.0f) { CustomEndCap = arrowCap, StartCap = LineCap.Square })
+            using (var graphics = Graphics.FromImage(image))
```
Comment (MarkusKgit) [inline MonkeyBot/Common/XYChart.cs]: E.g. here it would be perfectly fine by me to use var

File: MonkeyBot/CommandManager.cs (hunk 1 above)
Comment (BramEsendam) [reply MonkeyBot/CommandManager.cs]: I can revert the commit if you want?
Or is that what you said :P

Comment (MarkusKgit) [reply MonkeyBot/CommandManager.cs]: That would probably be the easiest way ;)

Comment (BramEsendam) [reply MonkeyBot/CommandManager.cs]: And too answer your first question, yes i used ReSharper and some other small tools for this :)

**Comment by BramEsendam** [comment]
I've pushed the revert :)

**Comment by MarkusKgit** [comment]
Thanks again ;) 
The codebase has grown over time and myself with it so if you spot any major code smells feel free to roast me :)

**Comment by BramEsendam** [comment]
Hahaha Will do! :P

**Pull request merged**